mathics/core/builtin.py

```diff
@@ -468,7 +472,15 @@ def get_functions(self, prefix="eval", is_pymodule=False):
                 if pattern is None:  # Fixes PyPy bug
                     continue
                 else:
-                    m = re.match(r"[(]([\w,]+),[)]\:\s*(.*)", pattern)
+                    # TODO: consider to use a more sophisticated
+                    # regular expression, which handles breaklines
+                    # more properly, that supports format names
+                    # with contexts (context`name) and be less
+                    # fragile against leaving spaces between the
+                    # elements.
+                    m = re.match(
+                        r"[(]([\w,]+),[ ]*[)]\:\s*(.*)", pattern.replace("\n", " ")
```

Comment:
I don't understand exactly which newlines in docstirng are getting removed.

If not, please show this in the pytest.

Reply:
Thanks for adding the comments in the pytest.